web/src/styles.css

```diff
@@ -111,15 +111,15 @@ line, path {
 }
 
 .country-panel{
-    overflow-y: auto;
+    overflow-y: scroll;
```

Comment:
scroll the country panel instead of the container

Reply:
Did you test this on desktop? I think there was a specific reason for scrolling the container versus the panel. I think it would be nice to redo the html structure for the panel if we have time, now it's a bit of a patchwork and the lack of consistency is making it difficult to bugfix.

Reply:
yes I tested it. I actually think it's better structured now. But let's review together